recipe/meta.yaml

```diff
@@ -54,13 +58,57 @@ requirements:
     - python
     - pytorch =*=cuda*
 
-test:
-  imports:
-    - flash_attn
-  commands:
-    - pip check
-  requires:
-    - pip
+outputs:
+  - name: {{ name|lower }}
+
+    files:
+      exclude:
+        - 'lib/python*/site-packages/fused_dense_lib.cpython*.so'
+        - 'lib/python*/site-packages/dropout_layer_norm.cpython*.so'
+
+    test:
+      imports:
+        - flash_attn
+      commands:
+        - pip check
+      requires:
+        - pip
+
+  - name: {{ fused_dense_lib_name|lower }}
+
+    files:
+      include:
+        - 'lib/python*/site-packages/fused_dense_lib.cpython*.so'
+
+    requirements:
+      run:
+        - {{ pin_subpackage(name|lower, exact=True) }}
```

Comment:
Ok, this is working locally. We can probably trim some of the output requirements, but I'm afraid to touch anything as it breaks easily and takes a long time to build. :)

Reply:
The conda-build logs tell you which host requirements are necessary for each output because it inspects the linkages between the binaries. There are warnings for overdepending and errors for missing dependencies.

Reply:
Yup cleaned up the output requirements, I think this is the minimal now.